src/com/maddyhome/idea/vim/group/SearchGroup.java

```diff
@@ -115,15 +110,13 @@ public int getLastDir() {
   public void resetState() {
     lastSearch = lastPattern = lastSubstitute = lastReplace = lastOffset = null;
     lastIgnoreSmartCase = false;
-    lastDir = 0;
+    lastDir = Direction.UNSET;
```

Comment:
@citizenmatt Don't you think we can remove UNSET value? It's used only here (I've checked `getLastDir` and "This method is used in AceJump integration plugin", it's okay to remove it). Wouldn't it make usage of this enum more convenient (no need to process UNSET value)?

Reply:
Sounds good. I'll remove `UNSET` and `getLastDir` as part of the next round of changes.

Reply:
Nono! Don't remove `getLastDir`! Sorry, I've written it incorrectly, I meant that it's okay to remove UNSET because `getLastDir` usage in AceJump integration plugin doesn't use `0` value. It would be safe just to have `1` and `-1`.
But `getLastDir`is still needed :)

Reply:
Ha ha! I read that wrong! I'll make sure to keep it. 😁